stock_picking_restrict_cancel_with_orig_move/__manifest__.py

```diff
@@ -3,7 +3,7 @@
 {
     "name": "Stock Picking Restrict Cancel with Original Moves",
     "summary": "Restrict cancellation of dest moves according to origin.",
-    "version": "11.0.1.0.2",
+    "version": "12.0.1.0.2",
```

Comment:
@kittiu When migrating, we reset the trailing version. So, '12.0.1.0.0'

Reply:
Yes, sorry, I didn't complete this one yet. It is in progress. I create PR just to say I will be doing it + to see travis log. 
Please help mark it "In Progress"

Reply:
Currently, there are 4 addons I am working on. Sorry for the confusion.

Reply:
No problem. Don't hesitate to mark 'WIP' in comment when creating it.